### ISSUE TITLE
grafana version 6.4.2 Table panel - unable to find application file 

### ISSUE THREAD

**Issue by lyny5101** [grafana/grafana#20075]
**What happened**:
After adding the table panel and writing more than 2 queries
(After selecting both measurement and value)
Changing the format as (when two or more queries have different format as)
grafana error :::: unable to fine application file

![image](https://user-images.githubusercontent.com/53931806/67751462-81107300-fa74-11e9-8372-fe1a1f0cf2f6.png)

 In the situation as picture, A Query or B Query change Format as 'Table'

**Environment**:
- Grafana version: 6.4.2
- Data source type & version:  influxDB 1.7.8
- OS Grafana is installed on: Ubunto 18.04.3-desktop-amd64
- User OS & Browser: Ubunto / firefox, window/ chrom,
- Grafana plugins: Table Panel - Native Plugin
- Others: 

![image](https://user-images.githubusercontent.com/53931806/67750723-2296c500-fa73-11e9-808e-571576039fd2.png)

**Comment by aocenas** [comment]
Thanks for reporting.

There seems to be some issue with combining the formats. The full stack trace:
```
transformers.ts:43 Uncaught TypeError: Cannot read property 'length' of undefined
    at Object.transform (transformers.ts:43)
    at transformDataToTable (transformers.ts:257)
    at TablePanelCtrl.push../public/app/plugins/panel/table/module.ts.TablePanelCtrl.render (module.ts:137)
    at TablePanelCtrl.push../public/app/plugins/panel/table/module.ts.TablePanelCtrl.onDataReceived (module.ts:133)
    at EventEmitter.emit (index.js:129)
    at Emitter.push../public/app/core/utils/emitter.ts.Emitter.emit (emitter.ts:11)
    at notifyAngularQueryEditorsOfData (QueryEditorRow.tsx:277)
    at QueryEditorRow.push../public/app/features/dashboard/panel_editor/QueryEditorRow.tsx.QueryEditorRow.componentDidUpdate (QueryEditorRow.tsx:100)
    at QueryEditorRow.<anonymous> (react-hot-loader.development.js:642)
    at commitLifeCycles (react-dom.development.js:17349)
```

Also there are possibility to get to mixed format without getting an error by different order of adding queries and switching the format which is weird.

Also interesting and possibly related is that the format is different if you switch back and forth:
<img width="855" alt="Screenshot 2019-10-30 at 11 08 43" src="https://user-images.githubusercontent.com/1014802/67848854-c6c94c00-fb05-11e9-8ed8-1c6c01bc0d4d.png">
<img width="860" alt="Screenshot 2019-10-30 at 11 08 50" src="https://user-images.githubusercontent.com/1014802/67848855-c6c94c00-fb05-11e9-99a7-a227993cc074.png">
<img width="847" alt="Screenshot 2019-10-30 at 11 08 55" src="https://user-images.githubusercontent.com/1014802/67848856-c6c94c00-fb05-11e9-953e-2054042835e5.png">

This was the same query just switching between formats 2 times.

**Comment by aocenas** [comment]
@dprokop Can you take a look to see how critical this is and whether to add it to backlog?

**Comment by mstaalesen** [comment]
We get the same in Grafana Enterprise 6.4.3

Only its with prometheus as datasource and its happening for the single stat panel too.

**Comment by lyny5101** [comment]
Could this bug be fixed?

Once the field key is not entered, no error occurs.
So, It is used by changing 'format as' without inputting field key.

However, if you change 'format as' after  forget  that inputed the field key, grafana make an error.

This erro Occurs on Chrome, firefox both.

**Comment by hugohaggmark** [comment]
Very interesting, I've looked back as far back as Grafana 6.0.0 and the same error is there too. But in previous versions it didn't always bubble up accept for the small red triangle in the panel header for some versions. 

![image](https://user-images.githubusercontent.com/562238/68218289-5c704a00-ffe4-11e9-8498-d0dc720e9a64.png)

So my question is have we ever supported different data transformers in the same `Table` panel? @ryantxu @torkelo do you know?

**Comment by torkelo** [comment]
No we have never supported table and time series queries at the same time in table panel.

But think this error is new

**Comment by mstaalesen** [comment]
Still happends in 6.4.4.

**Comment by torkelo** [comment]
Fix coming in 6.5

**Comment by lyny5101** [comment]
> 6.5 수정
Thank you for your attention to this matter.
I will confirm it when it is officially distributed.
